pkg/config/env_var.go

```diff
@@ -0,0 +1,86 @@
+package config
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/openshift/odo/pkg/util"
+)
+
+// EnvVar represents an enviroment variable
+type EnvVar struct {
+	Name  string `yaml:"Name"`
+	Value string `yaml:"Value"`
+}
+
+// EnvVarList represents a list of environment variables
+type EnvVarList []EnvVar
+
+// Merge merges the other EnvVarlist with keeping last value for duplicate EnvVars
+// and returns a new EnvVarList
+func (evl EnvVarList) Merge(other EnvVarList) EnvVarList {
+
+	var dedupNewEvl EnvVarList
+	newEvl := append(evl, other...)
+	uniqueMap := make(map[string]string)
+	// last value will be kept in case o
```

Comment:
missing words on the end of comment I think?

Reply:
resolved